ui/src/pages/Logs.tsx

```diff
@@ -1,3 +1,134 @@
-export default function Logs() {
-  return <div>This is the logs page!</div>;
+import { TextField } from '@mui/material';
+import * as React from 'react';
+import Paper from '@mui/material/Paper';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableHead from '@mui/material/TableHead';
+import TablePagination from '@mui/material/TablePagination';
+import TableRow from '@mui/material/TableRow';
+
+//Define Column interface
+interface Column {
+  id: string;
+  label: string;
+  minWidth?: number;
+  align?: 'right'; //feel free to change this to center instead
+}
+
+// defining columns array, readonly just means that it can't be altered after declaration
```

Comment:
NIT (nitpick): be consistent with how you comment. It's best practice to add a space between the `//` and your comment, like you do here. Also try to be consistent with your comments starting with an uppercase letter.